### ISSUE TITLE
Additional buttons do not work

### ISSUE THREAD

**Issue by WSCHUDO** [obaskly/midjourney_telegram_bot#3]
You got a cool bot =)

It turned out to launch the bot. I send him a request, he sends a general image consisting of 4 different images. 

When I choose /U 1,2,3,4 or /V 1,2,3,4 or /make_variations, then the image does not come

Help me please =)

**Comment by WSCHUDO** [comment]
The image itself marked _upscaled is not uploaded to the folder

**Comment by harryeffinpotter** [comment]
> You got a cool bot =)
> 
> It turned out to launch the bot. I send him a request, he sends a general image consisting of 4 different images.
> 
> When I choose /U 1,2,3,4 or /V 1,2,3,4 or /make_variations, then the image does not come
> 
> Help me please =)

it looked like in his video he just did U3 not /U 3

**Comment by WSCHUDO** [comment]
> > You got a cool bot =)
> > It turned out to launch the bot. I send him a request, he sends a general image consisting of 4 different images.
> > When I choose /U 1,2,3,4 or /V 1,2,3,4 or /make_variations, then the image does not come
> > Help me please =)
> 
> it looked like in his video he just did U3 not /U 3

Yes, but in reality the U3 command does not work without a prefix /

When I enter /U3 the action happens next, a new image starts to be generated but does not arrive in the telegram bot

**Comment by Juranik** [comment]
> You got a cool bot =)
> 
> It turned out to launch the bot. I send him a request, he sends a general image consisting of 4 different images.
> 
> When I choose /U 1,2,3,4 or /V 1,2,3,4 or /make_variations, then the image does not come
> 
> Help me please =)

Bro, please can you help, I beg you, how did you manage to set up the bot? Please help me
